build.bat

```diff
@@ -0,0 +1,51 @@
+@echo off
+MKDIR app src py-spy 2>NUL
+REM Download Build Tools for Visual Studio
+REM https://aka.ms/vs/17/release/vs_BuildTools.exe
+WHERE python
+IF %ERRORLEVEL% NEQ 0 (
+	ECHO python wasn't found. Attempting to install...
+	ECHO Shell will close once complete. Re-run build.bat to proceed
+	CALL .\dep\python-3.9.13-amd64.exe /passive PrependPath=1 Include_test=0
+)
+REM Get Python version
+FOR /f "tokens=1-2" %%i in ('python --version') do (
+	set PYTHON_VERSION=%%j
+	IF /i "%PYTHON_VERSION:~0,1%" == "3" (
+		ECHO Python version is valid
+	) ELSE (
+		ECHO Python 3 is required. Attempying to install...
+		ECHO Shell will close once complete. Re-run build.bat to proceed
+		.\deb\python-3.9.13-amd64.exe /passive PrependPath=1 Include_test=0
+	)
+)
+@echo Installed python version: %PYTHON_VERSION%
+IF EXIST .\py-spy\py-spy.exe (
+	ECHO Found py-spy.exe
+) ELSE (
+	ECHO Building py-spy executable...
+	CALL build-pyspy.bat
+)
+CD app
+
+WHERE pip
+IF %ERRORLEVEL% NEQ 0 (
+	ECHO pip wasn't found.
+	ECHO Install pip for current and re-run build.bat
```

Comment:
Install pip in the script here? (e.g https://phoenixnap.com/kb/install-pip-windows, via `get-pip.py`)

Reply:
Addressed it. You can install pip with `get-pip.py` on the build machine, for MVP also okay to assume pip is installed.